theano/gof/op.py

```diff
@@ -966,7 +966,7 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         if new_node is not None:
             node = new_node
 
-        if self._op_use_c_code:
```

Comment:
I think we should get rid of `_op_use_c_code` and just use theano.config.cxx since otherwise you could end up reloading a graph with `_op_use_c_code` set to True and no C compiler on the machine.  I really don't see why that has to be a property of the ops themselves.

But in any case I support the change above.